index.html

```diff
@@ -9,9 +9,9 @@
     <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
 </head>
 <body>
-<div class="page1">
+<section class="page1">
```

Comment:
[OPTIONAL] Let's try to stick to the font, font size, margins, and paddings as it is in the Figma Template.
The left padding in your design is too small that the text is very close to the left edge of the page. **Please, fix this I advise**.
![image](https://user-images.githubusercontent.com/45668340/148456072-c7acce75-15b0-49e0-a933-550df8c9fc04.png)
The font size of the header text is also not as big as that of Figma's.
## Figma's
![image](https://user-images.githubusercontent.com/45668340/148456183-4f44ffbf-5693-4bd5-99ec-cf185ecb882f.png)